integration-tests/pom.xml

```diff
@@ -32,6 +32,7 @@
         <module>jpa-mariadb</module>
         <module>jpa-h2</module>
         <module>jpa-mssql</module>
+        <module>hibernate-configuration</module>
```

Comment:
Was it really necessary to create an additional module instead of adding tests to the `jpa` module?

If we add a new module for each subfeature of each component, we will end up with a ton of modules.

There might be some good reason I miss though.

Reply:
I can move tests to the `jpa` module. The difference in tests is a new option. I didn't find how to run test with different profiles (one with the option, second without), so I splitted it into separate modules. Or I leave the tests with enabled option.